### ISSUE TITLE
object_usage_linter fails to parse inline function output

### ISSUE THREAD

**Issue by MichaelChirico** [r-lib/lintr#1917]
```r
lintr::lint("
foo <- function() s
", lintr::object_usage_linter())
```

Throws a warning about an internal error:

```
Warning message:
In parse_check_usage(fun, known_used_symbols = known_used_symbols,  :
  Possible bug in lintr: Couldn't parse usage message ‘<anonymous>: no visible binding for global variable 's'
’. Ignoring 1 usage warnings. Please report an issue at https://github.com/r-lib/
```
